### PR TITLE
Some cleanups and small things.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

**Comment by arokem** [comment]
This evolved to me trying to address some of the comments made by BFG.

**Comment by dhuppenkothen** [comment]
Wow, I just managed to actually read BFG's comments, and you've already implemented them! :D 
I'll go through this after my meetings this morning.

File: sections/what.tex [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 One criticism of hackathons is that they propel the ``geek'' stereotype and may present a barrier to creating an inclusive working environment, especially for individuals traditionally underrepresented in science and technology \cite{Decker2015}.
-Another problem is the competitive atmosphere of many industry hackathons, where teams actively compete for prizes.
-Because certain groups are more risk-averse than others, and risk-aversion traces demographics like gender, we urge caution when contemplating making an event competitive.
\ No newline at end of file
+This criticism needs to be actively addressed both through participant selection (see below), and by addressing the possibility that some participants may experience an ``imposter syndrome'' within such circumstances \cite{clance1978imposter}.
+Also, while many industry hackathons are competitive, with teams actively competing to solve the same problems for prizes that are awarded prizes, academic hackathons, and specifically the hack weeks that we have organized events are not explicitely competitive
```
Comment (dhuppenkothen) [inline sections/what.tex]: "for prizes that are awarded prizes" ?

**Review by dhuppenkothen — APPROVED** [review]

**Comment by dhuppenkothen** [comment]
Had five minutes to look at the changes after all. Just one typo, as far as I can see, otherwise I think these changes are great!

**Comment by arokem** [comment]
Typo fixed. I also added an acknowledgement to BFG for her comments on this.

**Pull request merged**